test/ui/viewJobRolesTest.ts

```diff
@@ -0,0 +1,18 @@
+import webdriver from 'selenium-webdriver';
+import { expect } from 'chai';
+import { By } from 'selenium-webdriver';
+
+describe('job-roles page', () => {
+
+    it('should display table of job roles', async () => {
+        const driver = new webdriver.Builder().forBrowser('chrome').build();
+
+        await driver.get(process.env.UI_TEST_URL + '/job-roles');
+
+        const jobRole = await driver.findElement(By.css('table'));
```

Comment:
you should get by ID rather than CSS

Reply:
This does get by ID, the method is for a CSS selector. The named get by ID method is deprecated and it says to use this.